earth_enterprise/rpms/opengee-fusion/post-install.sh

```diff
@@ -150,20 +144,20 @@ install_or_upgrade_asset_root()
     chmod 644 "$SYSTEMRC"
     chown "$GEFUSIONUSER:$GEGROUP" "$SYSTEMRC"
 
-    if [ "$IS_NEWINSTALL" == true ]; then
+    if [ ! -d "$ASSET_ROOT/.config" ]; then
         "$BASEINSTALLDIR_OPT/bin/geconfigureassetroot" --new --noprompt \
             --assetroot "$ASSET_ROOT" --srcvol "$SOURCE_VOLUME"
         chown -R "$GEFUSIONUSER:$GEGROUP" "$ASSET_ROOT"
     else
         # upgrade asset root -- if this is a master
         if [ "$IS_SLAVE" == false ]; then
             # TODO: Verify this logic -- this is what is defined in the
```

Comment:
Does this TODO still apply?

Reply:
I don't know what "the installer documentation" is, and why different versions of GEE have different file owners.  So, as far as I know, it still applies.

Reply:
I know it's not very clean, but I don't think that this should be holding up the merge.